.github/workflows/ci.yml

```diff
@@ -76,9 +76,9 @@ jobs:
       - name: Setup macOS environment
         if: runner.os == 'macOS'
         run: |
-          // Workaround for https://github.com/actions/virtual-environments/issues/1811
-          // Remove broken openssl and python packages, not used by Conscrypt builds
-          // TODO(prb): remove when no longer required.
+          # Workaround for https://github.com/actions/virtual-environments/issues/1811
+          # Remove broken openssl and python packages, not used by Conscrypt builds
+          # TODO(prb): remove when no longer required.
           brew uninstall openssl@2.0.2t
```

Comment:
I think this should be 1.0.2t

Reply:
doh! I thought it was failing because that package was now removed.
Who'd have thought such a simple fix would go through so many iterations, sorry.